VideoLocker/res/values/styles.xml

```diff
@@ -374,24 +371,19 @@
         <item name="android:layout_marginTop">@dimen/spinnerPopupBorder</item>
         <item name="android:paddingBottom">@dimen/spinnerPopupBorder</item>
         <item name="android:overlapAnchor" tools:targetApi="21">true</item>
-        <!-- TODO: Since AppCompatSpinner doesn't support backporting the
-             anchor overlapping feature currently in appcompat v22, we're
-             implementing our own custom solution of applying custom
-             vertical offsets in the relevant Spinners. Once we upgrade to
-             appcompat v23+, we can remove those workarounds and uncomment
-             this attribute so that it can be automatically handled by
-             AppCompatSpinner.
-        <item name="overlapAnchor">true</item>-->
+        <item name="overlapAnchor">true</item>
     </style>
 
+    <style name="edX.Widget.DropDownListView" parent="Widget.AppCompat.ListView.DropDown" />
+
     <!-- Spinner with the popup style -->
-    <style name="edX.Widget.ListPopupWindow" android:parent="Widget.AppCompat.ListPopupWindow">
+    <style name="edX.Widget.ListPopupWindow" parent="Widget.AppCompat.ListPopupWindow">
         <item name="android:popupBackground">@drawable/popup_menu_background</item>
         <item name="android:overlapAnchor" tools:targetApi="21">true</item>
```

Comment:
Yes, the base appcompat theme for `ListPopupWindow` is not defined to overlap the anchor. Even if it was though, it would be an implementation detail, and should be defined explicitly if it's part of the design requirements (as is the case currently).